content/finalist/ttjc@finalist@thesyedsufiyan/index.md

```diff
@@ -2,20 +2,19 @@
 title: Syed sufiyan
 type: "finalist"
 name: "syed sufiyan"
-portfolio: "https://syed-portfolio.netlify.app/"
-github_link: "https://github.com/theSyedsufiyan"
-linkedin_link: "https://www.linkedin.com/in/syed-sufiyans/"
-looking_for: "job/internship"
-image_link: "https://avatars0.githubusercontent.com/u/36100349?s=460&u=522f57940e95f59417f7ac6c6d27c9a84a7ae6ae&v=4"
-description: "Javascript | React | node.js | Html | Css"
-status: ''
----
+portfolio: "https://syedportfolio.netlify.app/"
+github-link: "https://github.com/theSyedsufiyan"
+linkedin-link: "https://www.linkedin.com/in/syed-sufiyans/"
+looking-for: "job/internship"
+image-link: "https://avatars0.githubusercontent.com/u/36100349?s=460&u=522f57940e95f59417f7ac6c6d27c9a84a7ae6ae&v=4"
+description: "Javascript | React | node.js | Html | Css | firebase | sass"
+
```

Comment:
You have changed the naming convention kindly have a look also I added a status tag please add that leave it blank by default 
```
status: ""
```
![image](https://user-images.githubusercontent.com/33570551/92293909-a4356a00-ef44-11ea-8eec-1d5c4a2f504a.png)

Reply:
yes i changed portfolio name , added 2 projects 
status keep empty.